daffodil-core/src/test/scala/org/apache/daffodil/core/general/TestTunables.scala

```diff
@@ -63,8 +64,8 @@ class TestTunables {
     val c2 = c.withTunable("maxSkipLengthInBytes", "2048")
     val pf2 = c2.compileNode(testSchema)
 
-    val dp1 = pf1.onPath("/")
-    var dp2 = pf2.onPath("/")
+    val dp1 = pf1.onPath("/").asInstanceOf[DataProcessor]
+    var dp2 = pf2.onPath("/").asInstanceOf[DataProcessor]
```

Comment:
Mentioned in another comment asking why we have withTunables on a scala data processor. If it's just for tests liek this, I would suggest we remove this test. It's just testing that we can change tunables on a DataProcessor, but that's probably not something we need to, or should, support.

Some tunables are used at compile-time, so changing them after compilation but before running could potentially lead to errors if the way things were compiled doesn't match with the way things were expected to run.